lib/model/database.dart

```diff
@@ -52,30 +49,42 @@ class UriConverter extends TypeConverter<Uri, String> {
   @override Uri fromSql(String fromDb) => Uri.parse(fromDb);
 }
 
-LazyDatabase _openConnection() {
-  return LazyDatabase(() async {
-    // TODO decide if this path is the right one to use
-    final dbFolder = await getApplicationDocumentsDirectory();
-    final file = File(path.join(dbFolder.path, 'db.sqlite'));
-    return NativeDatabase.createInBackground(file);
-  });
-}
-
 @DriftDatabase(tables: [Accounts])
 class AppDatabase extends _$AppDatabase {
   AppDatabase(super.e);
 
-  AppDatabase.live() : this(_openConnection());
-
   // When updating the schema:
   //  * Make the change in the table classes, and bump schemaVersion.
-  //  * Export the new schema and generate test migrations:
+  //  * Export the new schema and generate test migrations with drift:
   //    $ tools/check --fix drift
+  //    and generate database code with build_runner.
+  //    See ../../README.md#generated-files for more
+  //    information on using the build_runner.
   //  * Write a migration in `onUpgrade` below.
   //  * Write tests.
   @override
   int get schemaVersion => 2; // See note.
 
+  Future<void> _resetDatabase(Migrator m) async {
```

Comment:
Testing for this is blocked until #1172.

Why does something block testing this?

It seems like we should be able to test it like so:
* Make a database that has the current schema, plus some arbitrary extra column on accounts and some arbitrary extra table. And has a version greater than the current version.
* Attempt a migration.
* Check that the schema now is the correct one.

Reply:
I guess this function doesn't exactly meet the spec of `onUpgrade`, because it assumes `to` is the current latest schema version. That's what #1172 is about.

But that doesn't seem like it's an obstacle for testing that it correctly does the thing it's intended to do, which is to correctly migrate given that assumption.

Reply:
The test ended up being a bit more complicated, but it turned out quite doable. We can fix #1172 at the same time.